rotkehlchen/db/schema.py

```diff
@@ -598,6 +598,23 @@
 );
 """  # noqa: E501
 
+DB_CREATE_HISTORY_EVENTS = """
+CREATE TABLE IF NOT EXISTS history_events (
+    identifier TEXT NOT NULL PRIMARY KEY,
```

Comment:
The problem we are gonna end up having with this is that we will need to be updating this every time you want to edit the event. Same problem we got with other stuff such as ledger actions, trades etc.

We need to think more on a proper approach for this.